recipe-server/client/control_new/tests/components/common/test_EnvAlert.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import EnvAlert from 'control_new/components/common/EnvAlert';
+
+describe('<EnvAlert>', () => {
+  it('should work', () => {
+    const wrapper = () => shallow(<EnvAlert />);
+
+    expect(wrapper).not.toThrow();
+  });
+
+  describe('findPartialString', () => {
+    const testStrings = ['abc', 'def'];
+
+    it('should find a piece of a string amongst an array of strings', () => {
+      // abc is present in 'abc'
+      let result = EnvAlert.findPartialString('abc', testStrings);
+      expect(result).toBe(true);
+
+      // ef is present in 'def'
+      result = EnvAlert.findPartialString('ef', testStrings);
+      expect(result).toBe(true);
+
+      // xyz is not present
+      result = EnvAlert.findPartialString('xyz', testStrings);
+      expect(result).toBe(false);
+
+      // abcdef is not present
+      result = EnvAlert.findPartialString('abcdef', testStrings);
+      expect(result).toBe(false);
+    });
+  });
+
+  it('should have accurate production URLs', () => {
+    expect(EnvAlert.productionUrls).toEqual(['normandy-admin.prod.']);
```

Comment:
I agree, looking at it now. The idea was that I'd like some sort of red flag to be raised when the URLs change (if ever), to prevent some of the headaches of using hardcoded URLs.

An idea I was kicking around was defining the staging/prod URLs in the webpack config (using `webpack.DefinePlugin`), then replacing the hardcoded arrays with the webpack-global variable (e.g. `static prodUrls = PRODUCTION_URLS;`). How do you feel about that?